packages/react-components/src/components/VideoTile.tsx

```diff
@@ -374,23 +383,45 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
   raisedHandBackgroundColor = callingPalette.raiseHandGold;
 
   /* @conditional-compile-remove(reaction) */
-  const backgroundImageUrl = reaction !== undefined ? reactionEmoji.get(reaction?.reactionType) : '';
+  const backgroundImageUrl =
+    reaction !== undefined && reactionResources !== undefined
+      ? getEmojiResource(reaction?.reactionType, reactionResources)
+      : '';
   /* @conditional-compile-remove(reaction) */
   const currentTimestamp = new Date();
   /* @conditional-compile-remove(reaction) */
   const currentUnixTimeStamp = Math.floor(currentTimestamp.getTime() / 1000);
   /* @conditional-compile-remove(reaction) */
   const receivedUnixTimestamp = reaction ? Math.floor(reaction.receivedAt.getTime() / 1000) : undefined;
   /* @conditional-compile-remove(reaction) */
-  const canRenderReaction = receivedUnixTimestamp ? currentUnixTimeStamp - receivedUnixTimestamp < 3000 : false;
+  const canRenderReaction =
+    (receivedUnixTimestamp ? currentUnixTimeStamp - receivedUnixTimestamp < 3000 : false) &&
+    backgroundImageUrl !== undefined;
+  /* @conditional-compile-remove(reaction) */
+  if (canRenderReaction && backgroundImageUrl && backgroundImageUrl?.length > 0) {
+    fetch(`${backgroundImageUrl}`)
```

Comment:
this will happen on every render pass. Instead we should trigger this anytime the backgroundImageUrl changes. And we should also reset the setIsValidImageSource when the bcakgroundImageUrl changes

suggestion:
```
useEffect(() => {
  if (!backgroundImageUrl) {
    return;
  }

  fetch(`${backgroundImageUrl}`)
    .then((res) => setIsValidImageSource(res.ok));
    .catch((warning) => console.warn(`Sprite image for animation rendering failed with warning: ${warning}`));

  // reset setIsValidImageSource anytime backgroundImageUrl changes
  return (() => setIsValidImageSource(false));
}, [backgroundImageUrl]);
```

Reply:
Oh also add the warning if `!res.ok`

Reply:
Updated according to the review suggestion. Thanks :)